connect/runtime/src/main/java/org/apache/kafka/connect/storage/ConnectorOffsetBackingStore.java

```diff
@@ -140,6 +142,9 @@ public static ConnectorOffsetBackingStore withOnlyConnectorStore(
     private final Optional<KafkaOffsetBackingStore> connectorStore;
     private final Optional<TopicAdmin> connectorStoreAdmin;
 
+    private boolean isEOSEnabled;
```

Comment:
```suggestion
    private boolean exactlyOnce;
```
nit: similar to https://github.com/apache/kafka/blob/9b7f7e0fa09121e204540b46bd3d7340a0175e7d/connect/runtime/src/main/java/org/apache/kafka/connect/storage/KafkaOffsetBackingStore.java#L152